spec/main-spec.js

```diff
@@ -0,0 +1,26 @@
+describe("atom-ide-datatip", () => {
+
+    beforeEach(async () => {
+        jasmine.attachToDOM(atom.views.getView(atom.workspace))
+        // Activate activation hook
+        atom.packages.triggerActivationHook('core:loaded-shell-environment');
+        atom.packages.triggerDeferredActivationHooks();
+        // Activate package-deps
+        await atom.packages.activatePackage("busy-signal")
+        await atom.packages.activatePackage("atom-ide-markdown-service")
```

Comment:
It looks like you will need to install `atom-ide-markdown-service` as well. Only core packages are installed by default.